src/extra/libs/rlottie/lv_rlottie.h

```diff
@@ -25,6 +25,14 @@ extern "C" {
 /**********************
  *      TYPEDEFS
  **********************/
+typedef enum {
+    lv_rlottie_forward  = 0,
```

Comment:
Please use upper case fields.

I also suggest `LV_RLOTTIE_CTRL_...` and `lv_rlottie_ctrl_t` for consistency.

Reply:
Ok, done.